src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -13,12 +13,15 @@
             [trptcolin.versioneer.core :as versioneer]
             [clojure.java.jmx :as jmx]
             [puppetlabs.i18n.core :as i18n])
-  (:import (java.net URL)
+  (:import (java.net URI)
            (java.util.concurrent CancellationException)
            (java.lang.management ManagementFactory)
            (javax.management ObjectName)
            (clojure.lang IFn)))
 
+
+(set! *warn-on-reflection* true)
```

Comment:
No strong opinion - I'd been thinking that either we'd keep the file clean now that it's clean, or, we'd decide we didn't want to at some point later, and we'd remove the line then.  I don't think it should have any affect in production otherwise, offhand?

Reply:
(Oh, and to be clear, also happy to just take that out.)